content/articles/Introduction-to-Containers-and-Docker/index.md

```diff
@@ -0,0 +1,87 @@
+ 
+The API is used by the command-line interface to interact with the Docker daemon process. Docker objects such as containers, images, networks, and volumes are created and managed by the docker daemon.
+
+### Reasons why we use dockers
+
+Recently, Docker has grown highly popular in the software industry. It lets developers and system admins construct and operate applications in containers, thanks to the growing need for microservices and DevOps. Here are some of the factors that contribute to Docker's popularity:
+
+1. **It's Versatile**: You may execute a basic hello world program within a docker container, a web server such as Apache HTTP Server, or Nginx within a docker container, any heavy-weight application within a docker container, and even an operating system within a docker container. Although Docker does not advise running an operating system within a container, it is still possible to do so.
+
+2. **Containers in Docker are freely linked.**: A container is a self-contained entity with its resource quota, networking configuration, and other features that make it easier for system administrators to replace or update one container without impacting the others.
+
+3. **Highly secure**: Docker's Highly Secure feature ensures that containers are entirely isolated from outside processes.
+
+4. **Docker containers are small and portable**: Docker containers are lighter than virtual machines since they just establish separate userspace and use the underlying OS Kernel of the system on which they are run.
+
+5. **It is portable**: When we finish building an application on our local workstation and need to deploy it to production, we run into several problems with environment configuration. However, if you use Docker, you can describe the container setup procedures, and Docker will ensure that the environment is set up the same way every time you run it.
+
+### Docker Windows framework necessities and establishments
+
+Docker is an OS-level virtualization programming stage that assists clients with building and overseeing applications in the Docker climate with all its library conditions. Before we examine the idea of Docker Installation on Windows, let us first comprehend the arrangement/necessities to introduce Docker for Windows. 
+
+#### Docker Windows OS system requirements:
+
+For Dockers work area application to run as planned, your PC ought to meet the following requirements: 
+
+1. A Windows 10 64-bit OS. Also, the `Virtualization and Containers` Windows feature should be enabled.
+
+2. A minimum of 4GB system RAM is recommended. 
+
+> When Hyper-V is activated on Windows, any designed VirtualBox on your framework will quit working. 
+
+#### Docker Installation on Windows Step-By-Step. 
+
+1. Download the docker work area for windows executable record. To download the official file, [click here](https://docs.docker.com/docker-for-windows/install).
```

Comment:
Rephrase this sentence to "Download the `Docker Desktop for Windows` executable record"